libgnucash/backend/sql/escape.cpp

```diff
@@ -22,7 +22,7 @@
 
 /*
  * FILE:
- * esacpe.c
+ * escape.c
```

Comment:
The filename has become `escape.cpp`, though that's obviously not a spelling fix...

Reply:
Do you want this as a PR? https://github.com/Gnucash/gnucash/compare/master...jsoref:spelling-escape?expand=1

(I'm not going to make additional PRs w/o someone confirming...)